### DISCUSSION TITLE
Explicity mention that a media type object can be empty

### DISCUSSION THREAD

**Discussion by dret** [OAI/OpenAPI-Specification#3967] [Enhancements]
Would it be helpful to explicitly mention that it is permissible to have an empty media type object? We could even add an example with an API responding with an image where the media type itself is enough to specify the response type.

**Comment by dret** [comment]
Oh, for some context: https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#media-type-object

**Reply by handrews** [reply]
It doesn't have any required parameters so it definitely can be empty, and we actually already have two examples where it _is_ empty: https://spec.openapis.org/oas/latest.html#considerations-for-file-uploads

I really wanted to completely re-organize the data modeling and media type encoding areas to gather all of the information and examples in one place.  Right now it's scattered all over various Objects and introductory sections (and now appendices as well).  But that would be a lot of work.  We could consider it for 3.2, and we're already heading in that direction for 4.0 with the functional area organization.  So maybe we'll just do that in 4.0- I'd rather get 3.2 out quickly that do a massive reorganization.

**Comment by handrews** [comment]
I'm going to close this since we do have multiple examples of the media type being empty.
